scripts/benchmark.rb

```diff
@@ -1,22 +1,20 @@
 # frozen_string_literal: true
 
-require 'rubygems'
 require 'benchmark'
 
-N = 10_000
+NAMES_COUNT = 10_000
 
 def run(name)
-  require name
   Benchmark.bm do |rep|
-    rep.report("generating #{N} names (#{name} #{FFaker::VERSION})") do
-      N.times do
-        FFaker::Name.name
+    rep.report("generating #{NAMES_COUNT} names (#{name})") do
+      NAMES_COUNT.times do
+        name == 'ffaker' ? FFaker::Name.name : Faker::Name.name
       end
```

Comment:
ternary operator should go outside of loop.
```suggestion
      mod == 'ffaker' ? FFaker : Faker
      NAMES_COUNT.times { mod::Name.name }
```